modules/cnf-creating-nrop-cr-hosted-control-plane.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-creating-nrop-cr-hosted-control-plane_{context}"]
+= Creating the NUMAResourcesOperator custom resource for the hosted control plane
+
+When you have installed the NUMA Resources Operator, then create the `NUMAResourcesOperator` custom resource (CR) that instructs the NUMA Resources Operator to install all the cluster infrastructure needed to support the NUMA-aware scheduler on hosted control plane, including daemon sets and APIs.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the NUMA Resources Operator.
```

Comment:
Maybe it's worth adding a link to the the installation documentation:
https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/scalability_and_performance/cnf-numa-aware-scheduling#installing-the-numa-resources-operator_numa-aware